docs/metasploit-framework.wiki/dev/Setting-Up-a-Metasploit-Development-Environment.md

```diff
@@ -22,7 +22,7 @@ This guide has details for setting up both **Linux** and **Windows**.
 
 ### Linux
 
-1. Open a terminal on your Linux host and set up Git, build tools, and Ruby dependencies:
+* Open a terminal on your Linux host and set up Git, build tools, and Ruby dependencies:
```

Comment:
For context, what's the rationale on swapping from numbered items to bullet points? 👀

Reply:
It breaks the numbering on the page because the bullet points are all over the place.
If you open the page, it's constantly listed as `1. <something>`. 
So cleaner to just use bullet points instead.